wiz-broker/templates/secrets.yaml

```diff
@@ -15,6 +15,23 @@ type: Opaque
 data:
   connectorData: {{ include "wiz-broker.wizConnectorSecretData" . | fromYaml | toJson | b64enc | quote }}
 {{- end }}
+
+{{- if and .Values.global.broker.caCertificate.enabled .Values.global.broker.caCertificate.createSecret }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "wiz-broker.caCertificateSecretName" . | trim }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "wiz-broker.labels" . | nindent 4 }}
+  annotations:
+    {{- with .Values.global.broker.caCertificate.annotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+data:
+  caCertificate: {{ .Values.global.broker.caCertificate.pem | b64enc | quote }}
```

Comment:
maybe use stringData without  b64enc

Reply:
Tried playing around with it, but it'd be harder to get newlines right and might fail the yaml parsing. PEM files probably contain newlines...